language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
+COM_USERS_MFA_EDIT_FIELD_TITLE="Tytuł"
+COM_USERS_MFA_EDIT_FIELD_TITLE_DESC="Ty i administratorzy witryny zobaczycie tę nazwę na liście dostępnych metod uwierzytelniania wieloskładnikowego dla Twojego konta użytkownika. Nie podawaj żadnych informacji wrażliwych ani danych osobowych."
+COM_USERS_MFA_EDIT_PAGE_HEAD="Zmodyfikuj metodę uwierzytelniania wieloskładnikowego"
+COM_USERS_MFA_FIRSTTIME_INSTRUCTIONS_HEAD="Użyj uwierzytelniania wieloskładnikowego, aby zwiększyć bezpieczeństwo"
+COM_USERS_MFA_FIRSTTIME_INSTRUCTIONS_WHATITDOES="Oto jak to działa. Dodaj poniżej metodę uwierzytelniania wieloskładnikowego. Od teraz za każdym razem, gdy będziesz logować się do strony internetowej, będziesz proszony o skorzystanie z tej metody w celu dokończenia logowania. Nawet jeśli ktoś ukradnie Twoją nazwę użytkownika i hasło, nie będzie miał dostępu do Twojego konta na tej stronie internetowej."
+COM_USERS_MFA_FIRSTTIME_NOTINTERESTED="Nie pokazuj tego ponownie"
+COM_USERS_MFA_FIRSTTIME_PAGE_HEAD="Skonfiguruj uwierzytelnianie wieloskładnikowe"
+COM_USERS_MFA_INVALID_CODE="Uwierzytelnianie wieloskładnikowe nie powiodło się. Proszę spróbować ponownie."
+COM_USERS_MFA_INVALID_METHOD="Nieprawidłowa metoda uwierzytelniania wieloskładnikowego."
+COM_USERS_MFA_LBL_CREATEDON="Dodane: %s"
+COM_USERS_MFA_LBL_DATE_FORMAT_PAST="d F Y"
+COM_USERS_MFA_LBL_DATE_FORMAT_TODAY="H:i"
+COM_USERS_MFA_LBL_DATE_FORMAT_YESTERDAY="H:i"
+COM_USERS_MFA_LBL_LASTUSED="Ostatnio używane: %s"
+COM_USERS_MFA_LBL_PAST="%s"
+COM_USERS_MFA_LBL_TODAY="Dzisiaj, %s"
+COM_USERS_MFA_LBL_YESTERDAY="Wczoraj, %s"
+COM_USERS_MFA_LIST_DEFAULTTAG="Domyślne"
+COM_USERS_MFA_LIST_INSTRUCTIONS="Dodaj co najmniej jedną metodę uwierzytelniania wieloskładnikowego. Za każdym razem, gdy będziesz logować się do strony internetowej, zostaniesz poproszony o jego podanie."
+COM_USERS_MFA_LIST_PAGE_HEAD="Twoje opcje uwierzytelniania wieloskładnikowego"
+COM_USERS_MFA_LIST_REMOVEALL="Wyłącz"
+COM_USERS_MFA_LIST_STATUS_OFF="Uwierzytelnianie wieloskładnikowe <strong>nie jest włączone</strong>."
+COM_USERS_MFA_LIST_STATUS_ON="Uwierzytelnianie wieloskładnikowe jest włączone."
+COM_USERS_MFA_LOGOUT="Wyloguj"
+COM_USERS_MFA_MANDATORY_NOTICE_BODY="Włącz metodę uwierzytelniania wieloskładnikowego dla swojego konta użytkownika. Nie będziesz mógł kontynuować korzystania ze strony internetowej, dopóki tego nie zrobisz."
+COM_USERS_MFA_MANDATORY_NOTICE_HEAD="Uwierzytelnianie wieloskładnikowe jest obowiązkowe dla Twojego konta użytkownika"
+COM_USERS_MFA_SELECT_PAGE_HEAD="Wybierz metodę uwierzytelniania wieloskładnikowego"
+COM_USERS_MFA_USE_DIFFERENT_METHOD="Wybierz inną metodę"
+COM_USERS_MFA_VALIDATE="Potwierdź poprawność"
 COM_USERS_OR="lub"
 COM_USERS_PROFILE="Profil użytkownika"
-COM_USERS_PROFILE_CORE_LEGEND="Profile"
-COM_USERS_PROFILE_DEFAULT_LABEL="Twoje dane"
+COM_USERS_PROFILE_CORE_LEGEND="Profil"
+COM_USERS_PROFILE_DEFAULT_LABEL="Edytuj swój profil"
```

"Twoje dane" jest prostsze, 'profil" jest słowem spoza podstawowego słownika. Zostawmy, jak było.  Zwłaszcza, że służy także do przeglądania, niekoniecznie edycji danych. 
```suggestion
COM_USERS_PROFILE_DEFAULT_LABEL="Twoje dane"
```